content/ui-patterns/overlays.mdx

```diff
@@ -0,0 +1,450 @@
+
+</Caption>
+
+### Side sheets
+
+Side sheets slide from the right or left edges of the viewport. Left side sheets are reserved for global navigation drawers. Right side sheets are used for global actions, but can also be considered for quick previews in full-width pages, or configuration panels.
+
+Side sheets take the entire height of the viewport, and should be used sparingly. Don't use side sheets to present forms or flows that should be a page instead.
+
+## Responsive variants
+
+### Bottom sheets (specific to narrow viewports)
+
+A bottom sheet is a variant supported on narrow viewports made to facilitate reachability. Overlays that appear anchored or centered in large screens may adapt to appear as a bottom sheet on small screens.
+
+A bottom sheet variant can be used when the overlay content is too busy to appear anchored. A bottom sheet always dims the rest of the screen for focus. Tapping on the backdrop dismisses the overlay.
+
+Its height should always accommodate the content inside. If the content is too tall to fit in the viewport, the content region should scroll accordingly.
+
+On portrait mode, a bottom sheet takes up the full width of the viewport.
+
+<ImageContainer>
+  <img
+    width="680"
+    src="https://user-images.githubusercontent.com/293280/147170160-5392c649-0dcb-4c7c-adfc-2524a4a3915c.png"
+    alt=" "
+    style="background: none"
+  />
+</ImageContainer>
+<Caption>
+
+In order to work well in non-rectangular mobile screens and browser UI, bottom sheets may add extra padding inside the overlay, following [`env(safe-area-inset-*)`](https://developer.mozilla.org/en-US/docs/Web/CSS/env()) environment variables.
+
+</Caption>
+
+On landscape mode, a bottom sheet has a maximum width of 480px, centered horizontally.
+
+<ImageContainer>
+    <img
+        width="880"
+        src="https://user-images.githubusercontent.com/293280/147170826-73caa091-b71a-4296-beba-ccf3a2ccf9ab.png"
+        alt=" "
+        style="background: none"
+    />
+</ImageContainer>
+<Caption>Bottom sheet on landscape viewports.</Caption>
+
+<ImageContainer>
+    <img
+        width="500"
+        src="https://user-images.githubusercontent.com/293280/147170671-09e6002a-2622-41b6-afe5-f37f0890a6c3.png"
+        alt=" "
+        style="background: none"
+    />
+</ImageContainer>
+<Caption>Header indicator when no title and close button are present.</Caption>
+
+A bottom sheet without a title or close button gets a header indicator at the top. Slide it down to dismiss the overlay.
+
+### Full-screen (specific to narrow viewports)
+
+Full-screen overlays may be used to present content that needs all the available space on narrow viewports.
+
+Overlays that have one or more input fields should always use the full-screen variant.
+
+<!--
+## Navigation
+This section should cover in-overlay navigation such as drill-in scenarios, back button, etc.
+-->
+
+## Components
```

Comment:
I think this is fine to leave in, but its a bit in flux. I wouldn't be against removing this section completely for now.